trafix-codec/src/decoder/mod.rs

```diff
@@ -1,3 +1,5 @@
+//! Implementation of the Message decoder.
+
```

Comment:
We can try referencing the message here: 


```suggestion
//! Implementation of the [`Message`] decoder.
//!
//! [`Message`]: crate::message::Message
```